Service/Request/RequestFactory.php

```diff
@@ -22,6 +23,8 @@ class RequestFactory implements RequestFactoryInterface
 {
     use LoggerAwareTrait;
 
+    const METHODS_WITHOUT_BODY = ['GET', 'HEAD', 'OPTIONS'];
```

Comment:
`TRACE` are also valid method, that not require body

Reply:
Fixed